#!/usr/bin/env python3
PACKAGE = "main_controller"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# gen.add("section_number",    int_t,    0, "The section number to start the main_controller", 1,  0, 100)

gen.add("map_size", double_t, 0, "Squared map size", 3,  1, 10)

gen.add("max_scene_duration", int_t, 0, "Number of reactions the robot will perform at most", 10,  1, 100)

exit(gen.generate(PACKAGE, "main_controller", "main_controller"))
